### DISCUSSION TITLE
Undelegation

### DISCUSSION THREAD

**Discussion by dbwebapp** [BitCannaGlobal/WeedWallet#93] [General]
Undelegation process not working correctly , see video below : 

https://github.com/BitCannaGlobal/WeedWallet/assets/96944913/d7ac9479-8780-4fef-aadc-334324fcdc4b

**Comment by TJ-BitCanna** [comment]
Some first questions;
- Keplr was unlocked?
- Keplr did not have another pending transaction to be confirmed?

I only get a spinning waiting symbol when my Keplr is not unlocked. Can't reproduce your scenario yet

**Comment by TJ-BitCanna** [comment]
@artontray does this still occur?
